doc/man/man8/dmg.8

```diff
@@ -346,6 +346,22 @@ Show results of each SCM & NVMe device format operation
 .TP
 \fB\fB\-\-reformat\fR\fP
 Reformat storage overwriting any existing filesystem (CAUTION: destructive operation)
+.SS storage identify
+Blink the status LED on a given VMD device for visual SSD identification.
+
+\fBAliases\fP: i
+
+.SS storage identify vmd
+Quickly blink the status LED on a VMD NVMe SSD for device identification.
```

Comment:
Two identify commands with different blink frequency? I think single identify command is better.

Reply:
they are the same command. dmg storage identify has the single option "vmd"

Reply:
It does seem a bit awkward to add the extra subcommand:
`dmg storage identify vmd --uuid foo`
vs.
`dmg storage identify --uuid foo`

Do you plan on having different types of identifiable devices? Do we have to expose this to the user, or can we figure it out on the backend?

Reply:
@mjmac I originally had the command w/o the extra "vmd" subcommand, however I was trying to keep the syntax similar to the "dmg storage set nvme-faulty" and the "dmg storage replace nvme" commands. I am open to changing these commands to whatever you and @tanabarr believe would be most user friendly in the control plane.

Reply:
also, the only identifiable device will be VMD, similar to the storage replace nvme command.